test/test_backbone_utils.py

```diff
@@ -23,6 +22,23 @@ def test_resnet_fpn_backbone(backbone_name):
     y = resnet_fpn_backbone(backbone_name=backbone_name, pretrained=False)(x)
     assert list(y.keys()) == ["0", "1", "2", "3", "pool"]
 
+    with pytest.raises(ValueError):
+        y = resnet_fpn_backbone(backbone_name=backbone_name, pretrained=False, trainable_layers=6)
```

Comment:
Also you actually don't need to store the variable back in `y` E.g. https://github.com/pytorch/vision/blob/main/test/test_utils.py#L306 You can just call `resnet_fpn_backbone()`. (Every place) this would fix the linter error too.